pkg/ccl/cliccl/debug_backup.go

```diff
@@ -1,851 +0,0 @@
-	if err != nil {
-		return err
-	}
-	for _, path := range oldIncPaths {
-		priorPaths = append(priorPaths, backuputils.JoinURLPath(oldIncURI.Path, path))
-	}
-
-	// Search for incrementals in the new default location, i.e. the "/incrementals" subdir.
-	newIncURI := *uri
-	newIncURI.Path = backuputils.JoinURLPath(newIncURI.Path, backupbase.DefaultIncrementalsSubdir, subdir)
-	incStore, err := externalStorageFromURIFactory(ctx, newIncURI.String(), username.RootUserName())
-	if err != nil {
-		return errors.Wrapf(err, "connect to external storage")
-	}
-	defer incStore.Close()
-
-	newIncPaths, err := backupdest.FindPriorBackups(ctx, incStore, backupdest.OmitManifest)
-	if err != nil {
-		return err
-	}
-	for _, path := range newIncPaths {
-		priorPaths = append(priorPaths, backuputils.JoinURLPath(newIncURI.Path, path))
-	}
-
-	// List and report manifests found in all locations.
-	stores := make([]cloud.ExternalStorage, len(priorPaths))
-	rows := make([][]string, 0)
-	for i, path := range priorPaths {
-		uri.Path = path
-		stores[i], err = externalStorageFromURIFactory(ctx, uri.String(), username.RootUserName())
-		if err != nil {
-			return errors.Wrapf(err, "connect to external storage")
-		}
-		defer stores[i].Close()
-		manifest, _, err := backupinfo.ReadBackupManifestFromStore(ctx, nil /* mem */, stores[i],
-			nil, nil)
-		if err != nil {
-			return err
-		}
-		startTime := manifest.StartTime.GoTime().Format(time.RFC3339)
-		endTime := manifest.EndTime.GoTime().Format(time.RFC3339)
-		if i == 0 {
-			startTime = "-"
-		}
-		newRow := []string{uri.Path, startTime, endTime}
-		rows = append(rows, newRow)
-	}
-	cols := []string{"path", "start time", "end time"}
-	rowSliceIter := clisqlexec.NewRowSliceIter(rows, "lll" /*align*/)
-	return cli.PrintQueryOutput(os.Stdout, cols, rowSliceIter)
-}
-
-func runExportDataCmd(cmd *cobra.Command, args []string) error {
-	if debugBackupArgs.exportTableName == "" {
-		return errors.New("export data requires table name specified by --table flag")
-	}
-	fullyQualifiedTableName := strings.ToLower(debugBackupArgs.exportTableName)
-	manifestPaths := args
-	ctx := context.Background()
-	manifests := make([]backuppb.BackupManifest, 0, len(manifestPaths))
-	for _, path := range manifestPaths {
-		manifest, err := getManifestFromURI(ctx, path)
-		if err != nil {
-			return errors.Wrapf(err, "fetching backup manifests from %s", path)
-		}
-		manifests = append(manifests, manifest)
-	}
-
-	if debugBackupArgs.withRevisions && manifests[0].MVCCFilter != backuppb.MVCCFilter_All {
-		return errors.WithHintf(
-			errors.Newf("invalid flag: %s", cliflags.ExportRevisions.Name),
-			"requires backup created with %q", backupOptRevisionHistory,
-		)
-	}
-
-	endTime, err := evalAsOfTimestamp(debugBackupArgs.readTime, manifests)
-	if err != nil {
-		return errors.Wrapf(err, "eval as of timestamp %s", debugBackupArgs.readTime)
-	}
-
-	codec := keys.TODOSQLCodec
-	entry, err := backupccl.MakeBackupTableEntry(
```

Comment:
I think this function can go from backupccl now too, which is has been the "ugh, why am I maintaining this" edge lately.